### ISSUE TITLE
Ads not blocked anymore

### ISSUE THREAD

**Issue by YofarDev** [nopbreak/TwitchMod#29]
Twitch is still going hard to force us to watch ads, and unfortunatly your modded twitch doen't block them anymore.

I don't know if you have followed the way to block it on web browser, but the most recent trick (at the time I'm writing this) is one add-on for chrome/firefox with this code :

```
chrome.webRequest.onBeforeSendHeaders.addListener(
	({ requestHeaders }) => {
		for (const header of requestHeaders) {
			if (header.name.toLowerCase() === "user-agent")
				header.value = "Googlebot";
		}

		return { requestHeaders };
	},
	{ urls: ["*://*.ttvnw.net/*"] },
	["blocking", "requestHeaders"]
);
```

Maybe it can help you

**Comment by ali-layken** [comment]
I found that if you download yesterday's unstable and use adgaurd or blokada you get no ads

**Comment by nopbreak** [comment]
Fixed in 2.8
